utils/cronjobs_osgeo_lxd/cron_grass7_relbranch_build_binaries.sh

```diff
@@ -34,9 +34,11 @@
 #################################
 PATH=/home/neteler/binaries/bin:/usr/bin:/bin:/usr/X11R6/bin:/usr/local/bin
 
-GMAJOR=7
-GMINOR=8
-GPATCH=7 # required by grass-addons-index.sh
+BRANCH=`curl https://api.github.com/repos/osgeo/grass/branches | grep releasebranch_7 | grep '"name":' | cut -f4 -d'"' | sort -V | tail -n 1`
```

Comment:
The major version is hidden in this expression, so `releasebranch_7` needs to be some branch prefix variable or something.

Reply:
Right. Another option, to get rid of hardcoded versions completely is to fetch the _latest major version -1_ to run cron jobs for one version older than current.

That said, maybe also files should be renamed e.g.:
_cron_grass7_relbranch_build_binaries.sh_ to _cron_grass_legacy_relbranch_build_binaries.sh_
and
_cron_grass8_relbranch_build_binaries.sh_ to _cron_grass_latest_relbranch_build_binaries.sh_

I probably should also unify how versions are fetched across the different scripts...

Reply:
In case of renaming of sh-files, changes should be handed down to https://github.com/OSGeo/grass-addons/blob/grass8/utils/cronjobs_osgeo_lxd/README.md and https://github.com/OSGeo/grass-addons/blob/grass8/utils/cronjobs_osgeo_lxd/cron_job_list_grass

Reply:
Renaming has been done in #814. So, in essence, only https://github.com/OSGeo/grass-addons/blob/grass8/utils/cronjobs_osgeo_lxd/cron_job_list_grass still contains version numbers (log file storage into the right server directory).